src/frontend/screens/Library/components/InstallModal/index.tsx

```diff
@@ -136,8 +136,8 @@ export default React.memo(function InstallModal({
           setPlatformToInstall(e.target.value as InstallPlatform)
         }
       >
-        {availablePlatforms.map((p) => (
-          <option value={p.value} key={p.value}>
+        {availablePlatforms.map((p, i) => (
+          <option value={p.value} key={i}>
```

Comment:
is this change needed? in general something unique from the element is better than an index in a collection for the key (if, for any reason, the elements in the array change order -not sure if it can actually happen in this case-, the key points to an incorrect element, we've had issues with that in other places)

I'd argue to always use something that is unique from the current element unless there's a good reason not to

Reply:
They wont change the order since it is a hardcoded object. But after I have removed the useState related with the default platform, it was stuck on the first value. So I changed that, although other things might have fixed it and I just did not tested this change again. Anyway, I don't think this causes any harm like I said since it is a hardcoded object.